guides/controllers.md

```diff
@@ -273,7 +273,7 @@ defmodule HelloWeb.Router do
 ...
 ```
 
-If we go to [`http://localhost:4000/?_format=text`](http://localhost:4000/?_format=text), we will see %{"message": "this is some JSON"}.
+If we go to [`http://localhost:4000/?_format=json`](http://localhost:4000/?_format=json), we will see `%{"message": "this is some JSON"}`.
```

Comment:
I think it makes more sense to use text because json would typically require a separate pipeline. Maybe we use text and change what is rendered? 🤔

Reply:
That argument makes a lot of sense to me, but I'm not sure how best to rewrite the entire section to match that. Everything in the section is about rendering JSON, including defining a `PageJSON` module, etc, so I was just trying to make the last part of the example match the rest.

Does `put_view/2` work with a `text:` format? If so, is it sufficient to essentially replace `json` with `text` in the whole section and have it all be correct?

Reply:
Ah, good point. @chrismccord, your call. :)